### ISSUE TITLE
upgrading from 0.5.5 to 0.5.6 result in errror

### ISSUE THREAD

**Issue by maxandersen** [awestruct/awestruct#488]
```
[2015-10-01 13:33:26] ERROR ArgumentError: wrong number of arguments (3 for 1)
	/Users/max/code/jbosstools/jbosstools-website/_ext/features.rb:19:in `transform'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/pipeline.rb:83:in `block in apply_transformers'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/pipeline.rb:82:in `each'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/pipeline.rb:82:in `apply_transformers'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/engine.rb:382:in `generate_page'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/rack/generate.rb:35:in `call'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/rack-1.6.4/lib/rack/builder.rb:153:in `call'
	/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/rack-1.6.4/lib/rack/handler/webrick.rb:88:in `service'
	/Users/max/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/webrick/httpserver.rb:138:in `service'
	/Users/max/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/webrick/httpserver.rb:94:in `run'
	/Users/max/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/webrick/server.rb:294:in `block in start_thread'
```

the relevant line is https://github.com/jbosstools/jbosstools-website/blob/master/_ext/features.rb#L19

I'm not sure why @xcoulon added this but would be great to know why we break between 0.5.5 to 0.5.6 (no api changes?)  and how to fix it?

**Comment by maxandersen** [comment]
in addition I get funky errors like posts not having a date anymore resulting in:

```
Please see /Users/max/code/jbosstools/jbosstools-website/bin/blog/2014-12-11-what-we-should-do-in-2015-survey-results.html for more information
An error during rendering /Users/max/code/jbosstools/jbosstools-website/Users/max/code/jbosstools/jbosstools-website/bin/blog/2014-12-19-browsersim-custom-devices-support.adoc occurred.
While processing file /Users/max/code/jbosstools/jbosstools-website/bin/blog/2014-12-19-browsersim-custom-devices-support.adoc
An error occurred: undefined method `strftime' for nil:NilClass
/Users/max/code/jbosstools/jbosstools-website/_layouts/blog.html.haml:31:in `block in singleton class'
/Users/max/code/jbosstools/jbosstools-website/_layouts/blog.html.haml:-5:in `instance_eval'
/Users/max/code/jbosstools/jbosstools-website/_layouts/blog.html.haml:-5:in `singleton class'
/Users/max/code/jbosstools/jbosstools-website/_layouts/blog.html.haml:-7:in `__tilt_70307226657280'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/tilt-2.0.1/lib/tilt/template.rb:155:in `call'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/tilt-2.0.1/lib/tilt/template.rb:155:in `evaluate'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/tilt-2.0.1/lib/tilt/haml.rb:17:in `evaluate'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/tilt-2.0.1/lib/tilt/template.rb:96:in `render'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/handlers/base_tilt_handler.rb:134:in `rendered_content'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/handlers/layout_handler.rb:47:in `rendered_content'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/page.rb:143:in `rendered_content'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/handlers/layout_handler.rb:54:in `block in rendered_content'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/handlers/layout_handler.rb:40:in `call'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/handlers/layout_handler.rb:40:in `for_layout_chain'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/handlers/layout_handler.rb:51:in `rendered_content'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/page.rb:143:in `rendered_content'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/engine.rb:381:in `generate_page'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/engine.rb:364:in `block in generate_output'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:408:in `call'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:408:in `call_with_index'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:379:in `process_incoming_jobs'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:361:in `block in worker'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:354:in `fork'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:354:in `worker'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:345:in `block in create_workers'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:344:in `each'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:344:in `create_workers'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:301:in `work_in_processes'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/parallel-1.6.1/lib/parallel.rb:232:in `map'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/engine.rb:363:in `generate_output'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/engine.rb:84:in `run'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/cli/generate.rb:22:in `run'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/cli/invoker.rb:126:in `invoke_generate'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/lib/awestruct/cli/invoker.rb:50:in `invoke!'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/gems/awestruct-0.5.6/bin/awestruct:9:in `<top (required)>'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/bin/awestruct:23:in `load'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/bin/awestruct:23:in `<main>'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/bin/ruby_executable_hooks:15:in `eval'
/Users/max/.rvm/gems/ruby-2.2.1@jbosstools-website/bin/ruby_executable_hooks:15:in `<main>'
```

for every blog entry.

**Comment by LightGuard** [comment]
Should be fixed with e3e4cfd
